### ISSUE TITLE
aplayer progress drag event desktop browser.

### ISSUE THREAD

**Issue by VCodepp** [SevenOutman/vue-aplayer#84]
I get music from an API. the source URL of your music is like this: test.app.com/file?id=123451234123

the list is working. But when I push the progress bar, it reacts wrongly.
the song is starting from the beginning.

```vue
<div class="main-player" v-if="list.length">
    <music 
      :music="list[0]"
      :list="list"
      theme="#5b246d"
      listMaxHeight="90px"
      listFolded
    />
</div>
```

**Comment by SevenOutman** [comment]
@VCodepp This piece of code looks fine to me. A possible cause is your `list` somehow has changed. Check if it does.

**Comment by VCodepp** [comment]
Yes list is computed value. because I use VUEX  getters music list.

list value only works once 

```
computed: {
    list() {
      return this.$store.state.musicPlayer;
    },
    song() {
      return this.$store.state.musicPlayer[0];
    }
}
```

**Comment by SevenOutman** [comment]
@VCodepp The player starts from beginning probably because the `music` it receives has changed. The mechanism is similar to #16 . The `music` before and after may look equal but are actually two individual objects. Check the way you update your store and see if that can happen.

**Comment by VCodepp** [comment]
VUEX getters mainmusiclist before value length 0 after length 7. Is the problem here?

**Comment by VCodepp** [comment]
maybe music source extensions are not mp3?

like this ;

src : http://test.testapp.com/files?id=3GQ36VS9BPUynDG

**Comment by VCodepp** [comment]
The extensions in my music API are not mp3.

That's why the progress bar may be working incorrectly. But it's interesting to playing music.

I do not think it has a solution.
So I close this question.

**Comment by VCodepp** [comment]
Hi @SevenOutman this progress bar Working IE11 and Firefox but not working chrome/opera
